src/Plugin.php

```diff
@@ -71,6 +71,9 @@ public static function init() {
     // Add teaser image to RSS feeds.
     add_action('rss2_item', __NAMESPACE__ . '\Feed::rss2_item');
 
+    // Disable pingbacks.
+    add_action('pre_ping', 'pre_ping');
```

Comment:
https://developer.wordpress.org/reference/hooks/pre_ping/ is for outbound pings to other sites.  We want to disable inbound pings instead.